lib/daru/io/io.rb

```diff
@@ -275,6 +275,17 @@ def html_table_to_dataframe(table)
           order: table[:order],
           name: table[:name]
       end
+
+      CSV::Converters[:boolean_converter] = lambda do |f, _|
```

Comment:
In my opinion the convenience of modifying the hash beats the better practice. Can change it though if that's the consensus.